docs/tutorials/remote_soc.rst

```diff
@@ -0,0 +1,258 @@
+Building Your Own SoC
+=====================
+
+SiliconCompiler supports a "zero-install" remote workflow, which lets you write hardware designs on your local machine and build them on cloud servers. We are currently running a free public beta, and the :ref:`remote API <Server API>` is a simple collection of HTTP requests which we publish under an open-source license.
+
+You can use the remote flow to try SiliconCompiler or experiment with hardware design, without going through the trouble of building and installing a full suite of open-source EDA tools on your local machine. This tutorial will walk you through the process of building an ASIC containing one PicoRV32 RISC-V CPU core and 2 kilobytes of SRAM, on an open-source 130nm Skywater process node.
+
+Note that our public beta currently only supports open-source tools and PDKs. You can access the public beta without a signup or login, and it is designed to delete your data after your jobs finish, but it is not intended to process proprietary or restricted intellectual property! Please `review our terms of service <https://www.siliconcompiler.com/terms-of-service>`_, and do not submit IP which you are not allowed to distribute.
+
+You can find complete example designs which reflect the contents of this tutorial in the public SiliconCompiler repository, `with RAM <https://github.com/siliconcompiler/siliconcompiler/tree/main/examples/picorv32_ram>`_ and `without RAM <https://github.com/siliconcompiler/siliconcompiler/tree/main/examples/picorv32>`_.
+
+[TODO / CR feedback: Add a screenshot here (or at the bottom)?]
+
+Installing SiliconCompiler
+--------------------------
+
+You must have a recent version of Python3 and its Pip package manager in order to install SiliconCompiler. We recommend that Windows users download Python from `https://python.org/downloads/ <https://python.org/downloads/>`_, rather than the Microsoft Store.
+
+With that, you should be able to install SiliconCompiler directly from PyPi::
+
```

Comment:
@WRansohoff and @gadfort : Is the expectation that all these tutorials will be completely standalone? That is - they should be able to skip our Installation instructions and walk through these instructions? If that is not the case, we should point back to our installation instructions to avoid duplicate or out-of-sync instruction.

Reply:
Peter mentioned this in person today. I agree that the remote-specific instructions in the PicoRV32 tutorial should be moved to the remote section.